### PR TITLE
Fixed: Add version section to inventory

### LINKED ISSUE #104: Write section defining the Versions block
n/t

### PR DESCRIPTION
**Pull request by ahankinson**
Fixes #104

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -397,6 +397,88 @@ <h2>Manifest</h2>
 
         <section id="versions">
             <h2>Versions</h2>
+            <p>
+                An OCFL Object Inventory MUST include a block for storing versions. This block MUST have the
+                key of <code>versions</code> within the inventory, and it MUST be an array.
+            </p>
+            <p>
+                Within the <code>versions</code> array is a list of JSON objects of type <code>Version</code>.
```
Comment (zimeon) [inline index.html]: I think we want to make it plain that there must be at least one version, I suggest: 

> The <code>versions</code> array MUST be a list of one or more JSON objects of type <code>Version</code>.

File: index.html [hunk 2]
```diff
@@ -397,6 +397,88 @@ <h2>Manifest</h2>
...
+                        </p>
+                        <blockquote class="informative">
+                            <p>
+                                Non-normative note: The <a>logical state</a> of the object uses content-addressable
+                                techniques to map <a>logical file paths</a> to their bitstreams, as expressed in
+                                the manifest section of the inventory.
+                            </p>
+                            <p>
+                                Notably, the version state can be used to provide de-duplication of content within
+                                the object, by mapping multiple <a>logical file paths</a> to the same content digest
```
Comment (rosy1280) [inline index.html]: We made an editorial decision to use OCFL Object or content.

File: index.html [hunk 3]
```diff
@@ -397,6 +397,88 @@ <h2>Manifest</h2>
...
+                                in the manifest. Implementers may choose to use this functionality on an object-by-
+                                object basis. For example, they may choose to implement renaming (that is, changes
+                                in file <i>name</i>, but not file <i>content</i>) by pointing a new logical file path
+                                to the digest for content that was added in a previous version. Implementers may also,
+                                however, choose to accession a new file with a new name and same content as a new
+                                version.
```
Comment (rosy1280) [inline index.html]: We may want to clarify here by pointing to the Implementations Note section.  Right now this isn't very clear and presumably that section will make it more clear.

File: index.html [hunk 4]
```diff
@@ -397,6 +397,88 @@ <h2>Manifest</h2>
...
+                            </p>
+                            <p>
+                                Notably, the version state can be used to provide de-duplication of content within
+                                the OCFL Object, by mapping multiple <a>logical file paths</a> to the same content digest
+                                in the manifest. Implementers may choose to use this functionality on an case-by-
+                                case basis. For example, they may choose to implement renaming (that is, changes
+                                in file <i>name</i>, but not file <i>content</i>) by pointing a new logical file path
+                                to the digest for content that was added in a previous version. Implementers may also,
+                                however, choose to accession a new file with a new name and same content as a new
+                                version.
```
Comment (rosy1280) [inline index.html]: We should link to the Implementation Note section to further clarify this.

**Pull request merged**